pkg/config/testdata/static/config.toml

```diff
@@ -8,28 +8,24 @@ remote_config = "https://developer.fastly.com/api/internal/cli-config"
 ttl = "5m"
 
 [language]
-  [language.rust]
-  toolchain_constraint = ">= 1.49.0 < 2.0.0"
-  wasm_wasi_target = "wasm32-wasi"
+[language.rust]
+toolchain_constraint = ">= 1.49.0 < 2.0.0"
+wasm_wasi_target = "wasm32-wasi"
```

Comment:
I'm not certain what this test is supposed to be doing, but this should probably be `wasm32-wasip1`.